invenio_db/utils.py

```diff
@@ -69,3 +70,12 @@ def rebuild_encrypted_properties(old_key, model, properties):
         model.query.filter_by(**primary_key_fields).\
             update(update_values)
     db.session.commit()
+
+
+def create_alembic_version_table():
```

Comment:
I add this so that we can use `alembic (upgrade|downgrade)` after creating the db with `db create`